### PR TITLE
Reword informative prose in the Introduction and Scope sections

### PR DESCRIPTION
**Pull request by rdeltour**
The proposed changes are not critical, but I think they slightly clarify the "format vs. rule" difference.

### CODE REVIEW

File: act-rules-format.bs [hunk 1]
```diff
@@ -17,19 +17,19 @@ Introduction {#intro}
...
 
 
 Scope {#scope}
 ==============
 
-The ACT Rules Format scope is focused on the development of ACT Rules for the conformance testing of content created using web technologies, such as HTML, CSS, WAI-ARIA, SVG and more, including digital publishing. The ACT Rules Format, however, is designed to be technology agnostic, meaning it has no specific technology in mind. This also means that the ACT Rules Format could conceivably be used for other technologies.
+The ACT Rules Format defined in this specification is focused on the description of test rules applicable to to content created using web technologies, such as HTML, CSS, WAI-ARIA, SVG and more, including digital publishing. The ACT Rules Format, however, is designed to be technology agnostic, meaning it has no specific technology in mind. This also means that the ACT Rules Format could conceivably be used for other technologies.
```
Comment (rdeltour) [inline act-rules-format.bs]: The ACT Rules Format defined in this specification is focused on the description of rules applicable to content created using web technologies

File: act-rules-format.bs [hunk 2]
```diff
@@ -17,19 +17,19 @@ Introduction {#intro}
...
 
-Other accessibility requirements that may be applicable to web technologies should also be testable with ACT Rules. For example, the User Agent Accessibility Guidelines 2.0 is applicable to web-based user agents, for which ACT Rules could be developed, but other technologies can also be used to develop User Agents, which are not web-based. Because some accessibility requirements may be applicable to technologies other than web technologies, the ACT Rules Format may not be suitable for every part of these accessibility requirement.
+Other accessibility requirements that may be applicable to web technologies would also typically be testable with ACT Rules. For example, as the User Agent Accessibility Guidelines 2.0 is applicable to web-based user agents, ACT Rules could be developed that target these requirements. However, as other technologies which are not web-based can also be used to develop User Agents, the ACT Rules Format would not necessarily be suitable to every part of the User Agent Accessibility Guidelines 2.0.
```
Comment (rdeltour) [inline act-rules-format.bs]: Other accessibility requirements applicable to web technologies could also be testable with ACT Rules. 
For example, ACT Rules could be developed to test the conformance of web-based user agents to the User Agent Accessibility Guidelines.
However, the ACT Rules Format would not necessarily be suitable to describe tests for the conformance of a non-web-based user agent.

**Comment by rdeltour** [comment]
@maryjom I pushed the changes we just discussed at TPAC.

**Comment by moekraft** [comment]
From 11/30/17, fix grammar and spelling issues.

**Pull request merged**